spec/features/articles_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.feature "Articles", type: :feature do
+  before do
+    @category = FactoryBot.create(:category)
+  end
+
+  scenario "unsucessfully create a new article" do
+    article = FactoryBot.build(:article)
+
+    visit root_path
```

Comment:
You're visiting the `root_path` for each, so you could pull this out into `before(:each)` block, e.g.
```ruby
before(:each) do
....
end
```

And similarly for your comment spec

Reply:
Fixed in commit d8658a9d8f9d8dd7e6a416bd13cec8bdbf25a750